packages/application-components/src/components/custom-views/custom-views-selector/custom-views-selector.tsx

```diff
@@ -34,8 +41,68 @@ const Wrapper = styled.div<TWrapperProps>`
 `;
 
 const Container = styled.div`
-  background-color: ${designTokens.colorAccent98};
-  padding: ${designTokens.spacing25} ${designTokens.spacing60};
+  background: linear-gradient(
+      0deg,
+      ${designTokens.colorNeutral95},
+      ${designTokens.colorNeutral95}
+    ),
+    linear-gradient(
+      0deg,
+      ${designTokens.colorNeutral98},
+      ${designTokens.colorNeutral98}
+    );
+  padding: 7px ${designTokens.spacing20};
+  border: 1px solid ${designTokens.colorNeutral95};
+  border-radius: ${designTokens.borderRadius8};
+`;
+
+const hoverAndSelectedStateStyles = css`
+  box-shadow: ${designTokens.shadow1};
+  background-color: ${designTokens.colorSurface};
+  color: ${designTokens.colorPrimary30};
+`;
+
+type TCustomViewSelectorItemProps = {
+  selected?: boolean;
+  label: string;
+  onClick?: (
+    event: MouseEvent<HTMLButtonElement> | KeyboardEvent<HTMLButtonElement>
+  ) => void;
+};
+
+const CustomViewSelectorItem = (props: TCustomViewSelectorItemProps) => {
+  return (
+    <AccessibleButton
+      css={css`
+        display: flex;
+        align-items: center;
+        padding: ${designTokens.spacing20} ${designTokens.spacing25};
+        height: 30px;
```

Comment:
Nit: also here, why do we need to make an exception for this?

Can't we use the design token for buttons (medium)?

https://github.com/commercetools/ui-kit/blob/dc24278e712b45a08872cbf3608a3d36dabe74d0/design-system/materials/internals/definition.yaml#L971-L972

Reply:
Sounds good, just double checking with @NataliaSclipet 
<img width="912" alt="Screenshot 2023-11-28 at 10 10 03" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/890bf167-c4b5-471f-a4d7-8d8fe614a248">

Reply:
Can we keep this exception both for here and above please? I'm aware we are not using our design tokens, however these exceptions here has been defined and reviewed together with Sven and Simone.

Reply:
Thank you, that's fine with me 😊